client_data.hh

```diff
@@ -15,7 +15,6 @@
 
 enum class client_type {
     cql = 0,
-    thrift,
```

Comment:
I think there might be a problem here. If I understand correctly (I'm not an expert in this code), this code is used for "system.clients" which supposedly users use. Are these numbers (0,1,2) visible in this table? If they are, this patch changes their meaning - Alternator will switch from 2 to 1.
Or maybe we print strings, not numbers, to that table?

Reply:
good point. it's part of the (kind-of) on-disk protocol. will revert this change.